examples/data-sources/tidbcloud_dedicated_region/data-source.tf

```diff
@@ -0,0 +1,11 @@
+variable "region_id" {
+  type = string
```

Comment:
The default value for nullable is true.  Why here omit nullable and other two data source have.